server/mergin/sync/public_api_controller.py

```diff
@@ -216,6 +216,7 @@ def add_project(namespace):  # noqa: E501
         request.json["storage_params"] = {
             "type": "local",
             "location": generate_location(),
+            "public": request.json.get("public", False),
```

Comment:
@harminius I am bit confused here, what's the catch with this 'public' flag?

Reply:
I'm just looking at it now too. This has no sense. 
Marcel wanted something like this `request.json["public"] = request.json.get("public", False)`
Which I think is not needed as `public` is set to False in init when not passed.
Let me remove the line.